app/models/subnet.rb

```diff
@@ -363,6 +364,13 @@ def dns_servers
     [dns_primary, dns_secondary].select(&:present?)
   end
 
+  def masked_subnet_address
+    IPAddr.new(network).mask(mask)
+  rescue StandardError => e
+    logger.info "Network address #{network} cannot be masked: #{e}"
```

Comment:
Same q as in the other hunk: what do we need to do to make mask() fail (and IPAddr.new succeed!) but still use that address?